### ISSUE TITLE
No Mac version?

### ISSUE THREAD

**Issue by developermarine** [absoIute/Mega-Hack-v5#47]
You should make a version for Mac. Also, another idea? You could make a lite edition. Basically, a free version with a lot less features as a preview. Thanks, Miles.

**Comment by Eliaskandil** [comment]
Yea He Should Make A Version For Mac ><C

**Comment by simalei** [comment]
Version for Mac will require the rewrite from zero, so I don't think absolute will release MHv5 for Mac someday.

**Comment by developermarine** [comment]
> Version for Mac will require the rewrite from zero, so I don't think absolute will release MHv5 for Mac someday.

Yeah. :(
But, since then, Geode became a thing so

**Comment by developermarine** [comment]
> Yea He Should Make A Version For Mac ><C

Geode

**Comment by antonfdiaz** [comment]
> Yea He Should Make A Version For Mac ><C

Yeah, but he would need to rewrite almost all the code because the adresses and all that stuff is different on mac.
